### PR TITLE
Network ARM templates

### PR DESCRIPTION
**Pull request by YaronFruchtmann**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Parsers/ASimNetworkSession/ARM/ASimNetworkNotables/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Source agnostic ASIM Network Notables Normalization Parser
+
+This template deploys the ASIM NetworkSession schema parser for Source agnostic. The parser is a part of the Advanced SIEM Information Model.
```
Comment (oshezaf) [inline Parsers/ASimNetworkSession/ARM/ASimNetworkNotables/README.md:3]: Just a reminder that this my example for the need to update the generation script for readme files.

File: Parsers/ASimNetworkSession/ARM/ASimNetworkSession/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Source agnostic ASIM Network Sessions Normalization Parser
```
Comment (oshezaf) [inline Parsers/ASimNetworkSession/ARM/ASimNetworkSession/README.md:1]: Sessions in singular

File: Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# M365 Defender ASIM NetworkSessions Normalization Parser
```
Comment (oshezaf) [inline Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md:1]: Space between network and session (in singular)

File: Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+This template deploys the ASIM NetworkSessions schema parser for M365 Defender. The parser is a part of the Advanced SIEM Information Model.
```
Comment (oshezaf) [inline Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md:3]: Checked and the official name is "Microsoft Defender for Endpoint" (no 365, Endpoint in singular)

File: Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoftWindowsEventFirewall/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Windows Events Firewall ASIM NetworkSessions Normalization Parser
+
+This template deploys the ASIM NetworkSessions schema parser for Windows Events Firewall. The parser is a part of the Advanced SIEM Information Model.
```
Comment (oshezaf) [inline Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoftWindowsEventFirewall/README.md:3]: "Windows Events Firewall" sounds bad. I think "Windows Firewall Events"?

**Review by oshezaf — CHANGES_REQUESTED** [review]
Mostly about naming. So my historical mistakes. Still this time I reviewed :-)

File: Parsers/ASimNetworkSession/ARM/ASimNetworkNotables/README.md (hunk 1 above)
Comment (yaronMSFT) [reply Parsers/ASimNetworkSession/ARM/ASimNetworkNotables/README.md:3]: I did not generate readme files now. Duplicated the existing one. But I agree, we need to update the readme template  for the generator.

File: Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md (hunk 4 above)
Comment (yaronMSFT) [reply Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md:3]: I fixed the text for now, but changing the function name is more complicated

File: Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoftWindowsEventFirewall/README.md (hunk 5 above)
Comment (yaronMSFT) [reply Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoftWindowsEventFirewall/README.md:3]: I concur. Again, changing labels is the easy part.

**Review by oshezaf — APPROVED** [review]

**Pull request merged**